src/frontend/screens/Settings/components/Tools/index.tsx

```diff
@@ -23,15 +23,16 @@ export default function Tools() {
     return <></>
   }
 
-  type Tool = 'winecfg' | string
```

Comment:
Specifying `'stringLiteral' | string` as a type doesn't do anything, `string` always shadows the literal
So we might as well just use `string` below and skip this type definition